astropy/time/formats.py

```diff
@@ -731,17 +731,28 @@ def parse_string(self, timestr, subfmts):
             raise ValueError('Time {0} does not match {1} format'
                              .format(timestr, self.name))
 
+    @staticmethod
+    def _decode_if_needed(val, encoding='ascii'):
```

Comment:
Right now this `encoding` kwarg is orphaned, in the sense that it is not actually used and is not exposed in any way to the public API.  I'd rather hardwire for now and then add a kwarg IFF it is actually necessary.